docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -312,23 +312,16 @@ $ tsh login --proxy=teleport.example.com --user=alice --auth=local
 $ tctl auth sign --user=security-scanner --format=openssh --out=security-scanner --ttl=10h
 ```
 
-Here is a summary of variables and functions we used in this guide:
-
-<table>
-<tr>
-  <th>Variable or Function</th>
-  <th>Description</th>
-</tr>
-<tr>
-  <td>`user.spec.traits`</td>
-  <td>Access traits of local or SSO user.</td>
-</tr>
-<tr>
-  <td>`contains(list, var)`</td>
-  <td>Checks whether list contains variable</td>
-</tr>
-<tr>
-  <td>`equals(var, var)`</td>
-  <td>Checks whether one variable is equal another</td>
-</tr>
-</table>
+### Filter fields explained
+
+Explanation of the fields used in this guide for the `where` conditions.
+
+| Field                                             | Description                                                    |
+|---------------------------------------------------|----------------------------------------------------------------|
+| `user.spec.traits["<list name>"]`                 | the `list` of traits from a local or SSO user                  |
```

Comment:
I don't see `list` used literally within a configuration resource, so "the `list`" might not be the clearest description to use here. Is there another way to phrase this that might be more recognizable? In other words, where would a user know how to find the value to use here?

Reply:
actually, the doc says that  `user traits typically come from an external identity provider`, so this tells me this value will rarely change, so instead of `<list name>` i put the actual value of what it will typically be